plugins/categoryaslink/class.categoryaslink.plugin.php

```diff
@@ -122,22 +122,28 @@ function categoryUrl($category, $page = '', $withDomain = true) {
             return safeURL(val('RedirectUrl', $category));
         }
 
-        // Build URL.
```

Comment:
It looks like we went from having one if/else statement and one return statement to two separate if statements and three returns, which is maybe needlessly increasing the complexity here. For the sake of readability, I think you might be able to simplify "building the URL" into a single if/else-if/else statement and one return.